zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/schema/SchemaManagerIT.java

```diff
@@ -51,13 +45,38 @@
 @Testcontainers
 public class SchemaManagerIT {
 
-  private static final ObjectMapper MAPPER = new ObjectMapper();
   private static ExporterConfiguration config = new ExporterConfiguration();
+  private static ElasticsearchClient elsClient;
+  private static OpenSearchClient osClient;
+
+  @Container
+  private static final ElasticsearchContainer ELS_CONTAINER =
+      TestSupport.createDefeaultElasticsearchContainer();
+
+  @Container
+  private static final OpensearchContainer<?> OPENSEARCH_CONTAINER =
+      TestSupport.createDefaultOpensearchContainer();
+
   private IndexDescriptor index;
   private IndexTemplateDescriptor indexTemplate;
 
+  @BeforeAll
+  public static void init() {
+    config.getConnect().setUrl(ELS_CONTAINER.getHttpHostAddress());
+    elsClient = new ElasticsearchConnector(config.getConnect()).createClient();
+
+    config.getConnect().setUrl(OPENSEARCH_CONTAINER.getHttpHostAddress());
+    osClient = new OpensearchConnector(config.getConnect()).createClient();
+  }
+
   @BeforeEach
   public void refresh() throws IOException {
+    elsClient.indices().delete(req -> req.index("*"));
+    elsClient.indices().deleteIndexTemplate(req -> req.name("*"));
+    osClient.indices().delete(req -> req.index(config.getIndex().getPrefix() + "*"));
+    osClient.indices().deleteIndexTemplate(req -> req.name("*"));
```

Comment:
🔧 This will lead to double the time in the config phase of each test method. could we think of a way to only execute the clean-up for the client currently used for test? That would be nice to have if possible